internal/command/machine/start.go

```diff
@@ -57,28 +62,18 @@ func Start(ctx context.Context, machineID string) (err error) {
 		appName = appconfig.NameFromContext(ctx)
 	)
 
-	app, err := appFromMachineOrName(ctx, machineID, appName)
-	if err != nil {
-		return fmt.Errorf("could not make flaps client: %w", err)
-	}
-
-	flapsClient, err := flaps.New(ctx, app)
-	if err != nil {
-		return fmt.Errorf("could not make flaps client: %w", err)
-	}
-
-	machine, err := flapsClient.Start(ctx, machineID)
+	machine, err := flaps.FromContext(ctx).Start(ctx, machineID)
 	if err != nil {
 		switch {
-		case strings.Contains(err.Error(), "not found") && appName != "":
-			return fmt.Errorf("machine %s was not found in app %s", machineID, appName)
+		case strings.Contains(err.Error(), "not found"):
+			return fmt.Errorf("machine %s was not found in app '%s'", machineID, appName)
 		default:
 			return fmt.Errorf("could not start machine %s: %w", machineID, err)
 		}
 	}
```

Comment:
Could this reuse `rewriteMachineNotFoundErrors`?

Reply:
Just updated `start`, `stop`, `destroy` and `kill` to use it.